articles/quickstart/native/ios-swift/getting-started.md

```diff
@@ -4,24 +4,16 @@ description: Initial steps to integrate Auth0 in an iOS Swift application.
 budicon: 448
 ---
 
-This multistep quickstart guide will walk you through managing authentication in your iOS Swift apps with Auth0.
+This quickstart guide walks you through managing authentication with Auth0 in your iOS Swift apps.
 
-## About the Sample Applications
+## Sample Projects
 
-Each step within this quickstart guide (starting with **Login**) contains a downloadable sample which demonstrates the topic being covered. Within each of these samples, you will find an appropriate application to demonstrate the features discussed in the chapter.
+Each tutorial in this guide includes a link to a sample project showing the features described in the article. You can find all the samples in the [iOS Swift samples folder](https://github.com/auth0-samples/auth0-ios-swift-sample).
 
-## Auth0 Dependencies
+## Dependencies
 
-Each tutorial will require you to use the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit.
+Each tutorial uses the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit. It is a toolkit that lets you communicate with many of the basic [Auth0 API](/api/info) functions in a neat way.
```

Comment:
This sounds odd, have a look at the description in the Auth0.swift SDK https://github.com/auth0/Auth0.swift#auth0swift